proto/regen/ecocredit/basket/v1/query.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
+
+import "google/api/annotations.proto";
+import "regen/ecocredit/basket/v1/state.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+// Msg is the regen.ecocredit.basket.v1beta1 Query service.
+service Query {
+
+    // Basket queries one basket by denom.
+    rpc Basket(QueryBasketRequest) returns (QueryBasketResponse) {
+        option (google.api.http).get =
+            "/regen/ecocredit/v1alpha1/baskets/{basket_denom}";
+    }
+
+    // Baskets lists all baskets in the ecocredit module.
+    rpc Baskets(QueryBasketsRequest) returns (QueryBasketsResponse) {
+        option (google.api.http).get =
+            "/regen/ecocredit/v1alpha1/baskets";
```

Comment:
I  think we should change this to  either: 
1. /regen/ecocredit/v1/baskets
2. /regen/ecocredit/baskets/v1

Reply:
i'll do 1. to keep in alignment with how we usually do it

Reply:
2 is more consistent

Reply:
okay edited to be more aligned with the package paths, how does it look now?

Reply:
let's do 2 please, it lines up with the proto package path and i think will be the general way we do this

Reply:
LGTM now 👍